docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -5,6 +5,9 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+[float]
```

Comment:
This should list all the files that have been changed, but does not. I will need to look into this.